lib/s3-s3.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * AWS.S3-like behavior with failover to a secondary S3 location
+ */
+
+'use strict';
+
+// Used in init() below, this function mimics AWS.S3
+var s3 = function (primaryS3, secondaryS3) {
```

Comment:
Because... I changed this too much and didn't think about it?  :)  That's a great point, will change this now.